ftl/core/importing.ftl

```diff
@@ -72,10 +72,10 @@ importing-note-updated =
         [one] { $count } note updated
        *[other] { $count } notes updated
     }
-importing-processed-media-file =
```

Comment:
This one wasn't introduced by this PR, so shouldn't it be kept around and garbage collected later?